.github/CHANGELOG.md

```diff
@@ -40,6 +40,21 @@ This release contains contributions from (in alphabetical order):
 
 ### New features since last release
 
+* Job lists can now be filtered by status as well.
+  [(#30)](https://github.com/XanaduAI/xanadu-cloud-client/pull/30)
+
+  Using the CLI:
+
+  ```bash
+  xcc job list --status '<status>'
+  ```
+
+  Using the Python API:
+
+   ```python
+  xcc.Job.list(connection, status='<status>')
```

Comment:
Same as above, except we should also use `"` instead of `'` as per the Python conventions in this project.
```suggestion
  xcc.Job.list(connection, status="<Status>")
```